pkg/service/app/handler.go

```diff
@@ -670,13 +695,41 @@ func (p *Server) ModifyAppVersion(ctx context.Context, req *pb.ModifyAppVersionR
 
 		_, err = attachmentManagerClient.ReplaceAttachment(ctx, &pb.ReplaceAttachmentRequest{
 			AttachmentId:      version.PackageName,
-			AttachmentContent: attachmentContent,
+			AttachmentContent: content,
```

Comment:
replace attachment with package_files

Reply:
what means?
three steps to replace attachment with package_files 
1.use proxy to get whole package files(m1 = map[string][]byte);
2.use m2 = map[string][]byte passed in to replace the old one, eg.m1["config.json"]=m2["config.json"];
3.save m1

Reply:
> replace attachment with package_files

what means?
three steps to replace attachment with package_files
1.use proxy to get whole package files(m1 = map[string][]byte);
2.use m2 = map[string][]byte passed in to replace the old one, eg.m1["config.json"]=m2["config.json"];
3.save m1

or,you means that should add a rpc named ReplaceAttachmentWithPackageFiles to handle?